generator/src/html/script/Main.hx

```diff
@@ -18,9 +19,34 @@ class Main {
 		// parse and locate myselft
 		assert(tocData != null, "toc data is missing");
 		var toc = J(JQuery.parseHTML(tocData));
-		assert(document.URL.startsWith(document.baseURI));
-		var myUrl = document.URL.replace(document.baseURI, "").replace(window.location.hash, "");
-		if (myUrl == "")
+        
+        var baseURI = null;// document.baseURI ;
+        //FIXME IE workaround for the lack of document.baseURI 
+        if (baseURI == null) {
```

Comment:
I think it would be better not to branch here, because the workaround implementation isn't a one liner.  It's probably safer to have it as the only implementation.

Any runtime overhead is expected to be negligible (for the browsers that don't need it), and it's best to know as soon as possible when we break this.